bin/secrets/common.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
I was getting `In POSIX sh, echo flags are not supported. [SC2039]` in the `echo -n ...` line, btw shellcheck is great and integrates very well with emacs